spec/features/visits/03_users_can_edit_a_visit_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "rails_helper"
+
+feature "users can edit visits within a vacation" do
+  scenario "signed in user can edit a visit they created" do
+    visit "/"
+    click_link "Sign in with Google"
+    park = FactoryGirl.create(:park, name: "Kenai Fjords")
+    vacation = FactoryGirl.create(:vacation, user: User.last,
+      start_date: Date.new(2016,5,1), end_date: Date.new(2016,5,9))
+    visit = Visit.create(user: User.last, vacation: vacation, park: park,
+      start_date: vacation.start_date + 1, end_date: vacation.end_date - 1)
+
+    FactoryGirl.create(:park, name: "Badlands")
+    visit "/vacations/#{vacation.id}"
```

Comment:
There are path helpers so you don't have to define the routes everywhere, run `rake routes` to see all the available ones. In this case I think it's `vacations_path(vacation)`